start-dependencies.sh

```diff
@@ -51,6 +56,9 @@ if [ "$INSTALL_LOKI" == "true" ]; then
     --set "loki.storage.type=filesystem" \
     --set "loki.rulerConfig.storage.type=local" \
     --set "loki.auth_enabled=false" \
+    --set "loki.commonConfig.replication_factor=1" \
+    --set "loki.commonConfig.ring.instance_addr=127.0.0.1" \
+    --set "loki.commonConfig.ring.kvstore.store=inmemory" \
     --set "monitoring.selfMonitoring.enabled=false" \
     --set "monitoring.selfMonitoring.grafanaAgent.installOperator=false" \
```

Comment:
I would include all the disabling rules for `monitoring`. If you use `helm template` you can see why. It is better to avoid letting the chart create things with Prometheus CRDs (service monitors, alert rules, and so on) or create/start unnecessary things. I would add (as discussed):

```
  --set monitoring.serviceMonitor.enabled=false \
  --set monitoring.dashboards.enabled=false \
  --set monitoring.rules.enabled=false \
  --set monitoring.alerts.enabled=false \
  --set monitoring.selfMonitoring.lokiCanary.enabled=false \
```
For similar reasons, disable the Nginx Gateway as we will be reaching Loki directly:
```
  --set gateway.enabled=false \
```

Reply:
Done. See ed500dfe670164b5f0c1f11a59a76591b3f7e5a3. I reordered slightly to keep all of the `monitoring.selfMonitoring` values together.